Makefile

```diff
@@ -1,19 +1,28 @@
 all: lint build unit
 
 OUTPUT_DIR="build/_output"
+# Set the go mod vendor flags, if they're not already set
+GOFLAGS? = $(shell go env GOFLAGS)
+ifeq "$(findstring -mod=vendor,$(GOFLAGS))" "-mod=vendor"
+GO_MOD_FLAGS ?=
+else
+GO_MOD_FLAGS ?= -mod=vendor
+endif
```

Comment:
@ravisantoshgudimetla what happens if `GOFLAGS=-mod=` before `make build` is executed? I am observing locally that it remains as is and does not become `-mod=vendor`.